src/link/link_info/ipvlan.rs

```diff
@@ -165,3 +167,43 @@ impl From<IpVlanMode> for u16 {
         }
     }
 }
+
+const IPVLAN_FLAG_BRIDGE: u16 = 0;
+const IPVLAN_FLAG_PRIVATE: u16 = 1;
```

Comment:
Please use kernel constant name to helping other developers to validate.

In this case, it should be `IPVLAN_F_PRIVATE`

Reply:
done. thanks